docs/src/pages/sveltekit.mdx

```diff
@@ -0,0 +1,331 @@
+import { Callout, Steps } from "nextra-theme-docs";
+
+# Getting Started with SvelteKit
+
+Make sure you
+[install the packages and get your env variables first](/getting-started)
+
+## Package Setup
+
+<Steps>
+
+### Install the packages
+
+import { Tab, Tabs } from "nextra-theme-docs";
+
+{/* prettier-ignore-start */}
+
+<Tabs items={["npm", "yarn", "pnpm"]}>
+  <Tab>
+    ```bash copy 
+    npm install uploadthing @uploadthing/svelte
+    ```
+  </Tab>
+  <Tab>
+    ```bash copy 
+    yarn add uploadthing @uploadthing/svelte
+    ```
+  </Tab>
+  <Tab>
+    ```bash copy 
+    pnpm install uploadthing @uploadthing/svelte
+    ```
+  </Tab>
+</Tabs>
+
+{/* prettier-ignore-end */}
+
+### Add env variables
+
+```bash copy
+UPLOADTHING_SECRET=... # A secret key for your app (starts with sk_live_)
+UPLOADTHING_APP_ID=... # Your app id
+```
+
+<Callout type="warning">
+  If you don't already have an UploadThing secret key, [sign
+  up](/create-account) and create one from the
+  [dashboard!](https://uploadthing.com/dashboard)
+</Callout>
+
+### Define process in vite.config
+
+UploadThing internals use process.env which is not defined in vite, so it is
+necessary to add the following to your vite.config.
+
+```ts copy filename="vite.config.ts"
+import { sveltekit } from "@sveltejs/kit/vite";
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  plugins: [sveltekit()],
+  define: {
+    "process.env": {},
+  },
+});
+```
+
+</Steps>
+
+## Set Up A FileRouter
+
+<Steps>
+### Creating your first FileRoute
+
+All files uploaded to UploadThing are associated with a FileRoute. Think of a
+FileRoute similar to an endpoint, it has:
+
+- Permitted types ["image", "video", etc]
+- Max file size
+- (Optional) `middleware` to authenticate and tag requests
+- `onUploadComplete` callback for when uploads are completed
+
+```ts copy filename="src/lib/server/uploadthing.ts"
+import { createUploadthing } from "uploadthing/sveltekit";
+import type { FileRouter } from "uploadthing/sveltekit";
+
+const f = createUploadthing();
+
+export const uploadRouter = {
```

Comment:
Can you make this router match the other adapters? We have a specific page for fileroutes which shows more detail so the introduction/quickstart only needs the basic